hnormalise.cabal

```diff
@@ -1,5 +1,5 @@
 name:                hnormalise
-version:             0.4.6.0
+version:             0.4.7.0
 synopsis:            Log message normalisation tool producing structured JSON messages
 description:         Log message normalisation tool producing structured JSON messages
 homepage:            https://github.com/itkovian/hnormalise#readme
```

Comment:
next time you choose a license for a project, can you discuss this first in  a meeting. also, please change the copyright, it belongs to ugent.

Reply:
Since I've worked on it during work hours, I guess it does now

Reply:
feel free to add yourself as author though 😄